js/README.md

```diff
@@ -83,6 +84,31 @@ console.log(table.toString());
 */
 ```
 
+## Create a Table from JavaScript arrays
+
+```es6
+const fields = [{
+        name: 'precipitation',
+        type: { name: 'floatingpoint', precision: 'SINGLE'},
+        nullable: false, children: []
+    }, {
+        name: 'date',
+        type: { name: 'date', unit: 'MILLISECOND' },
+        nullable: false, children: []
+    }];
+const rainAmounts = Array.from({length: LENGTH}, () => Number((Math.random() * 20).toFixed(1)));
+const rainDates = Array.from({length: LENGTH}, (_, i) => Date.now() - 1000 * 60 * 60 * 24 * i);
+
+const LENGTH = 2000;
+const rainfall = arrow.Table.from({
+  schema: { fields: fields },
+  batches: [{ 
+    count: LENGTH,
+    columns: [
+      {name: "precipitation", count: LENGTH, VALIDITY: [], DATA: rainAmounts },
+      {name: "date",          count: LENGTH, VALIDITY: [], DATA: rainDates } ] }] })
+```
+
```

Comment:
nitpicky, but this won't actually work when copy-pasted because `LENGTH` is defined after the arrays.

It would be really nice if we could finish [ARROW-2766](https://issues.apache.org/jira/browse/ARROW-2766) to make this example much cleaner. Inferring the types from arrays is difficult though.. so we could do a subset of that functionality that takes a list of already created vectors and a list of names for them. We can _almost_ do this already, I can write up a PR that cleans it up.

Reply:
Yeah, let's get this out and can add more later. Both code, tutorials, and communty links can all be improved :) I'm still trying to figure out how to make a graphistry.slack.com #arrow signup page link...

Reply:
Sounds good - I can update the docs when I put that PR up

Reply:
See #2322 

Any objections to going ahead and merging this PR?